pkg/reconciler/autoscaling/kpa/scaler.go

```diff
@@ -116,7 +120,12 @@ func newScaler(ctx context.Context, psInformerFactory duck.InformerFactory, enqu
 func paToProbeTarget(pa *pav1alpha1.PodAutoscaler) string {
 	svc := pkgnet.GetServiceHostname(pa.Status.ServiceName, pa.Namespace)
 	port := networking.ServicePort(pa.Spec.ProtocolType)
-	return fmt.Sprintf("http://%s:%d/", svc, port)
+
+	// Safe to ignore error since we know the string is a valid URL
+	httpDest, _ := url.Parse(fmt.Sprintf("http://%s:%d/", svc, port))
+	httpDest.Path = path.Join(httpDest.Path, probePath)
+
+	return httpDest.String()
 }
```

Comment:
Perhaps here we can just 
```suggestion
     return fmt.Sprintf("http://%s:%d/healthz", svc, port)
}
```

Reply:
Done!